File: Java/Clase01/MundoPC/src/ar/com/system2025/mundopc/DispositivoEntrada.java
package ar.com.system2025.mundopc;

public class DispositivoEntrada {

    private String tipoEntrada;
    private String marca;

    public DispositivoEntrada(String tipoEntrada, String marca){
        this.tipoEntrada = tipoEntrada;
        this.marca = marca;
    }

    public String getTipoEntrada() {
        return tipoEntrada;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }

    public void setTipoEntrada(String tipoEntrada) {
        this.tipoEntrada = tipoEntrada;
    }

    @Override
    public String toString() {
        return "\n Marca:" + getMarca() + "\n Tipo Entrada: "+ getTipoEntrada();
    }

}

File: JavaScript/Tarea JS Clase 1/DispositivoEntrada.js
export class DispositivoEntrada {
    constructor(tipoEntrada, marca) {
        this._tipoEntrada = tipoEntrada;
        this._marca = marca;
    }

    get tipoEntrada() {
        return this._tipoEntrada;
    }

    set tipoEntrada(value) {
        this._tipoEntrada = value;
    }

    get marca() {
        return this._marca;
    }

    set marca(value) {
        this._marca = value;
    }

    toString() {
        return `${this._tipoEntrada} de marca ${this._marca}`;
    }
}